### PR TITLE
Follow up of the "bye bye" R2D2 PR: More marine obs in the ctests

### PR DESCRIPTION
**Pull request by guillaumevernieres**
- I moved all the untested obs `yaml` under `obs/old`, assuming that they probably are all wrong since the removal of R2D2
- I added (fake) AMSR2 icec cdl files
- Same as above but for ADT, also skipping the concatenation post ioda processing and concatenating with the new ioda converters
- I've made an effort to try to use the provider's file naming convention, but do check.

### CODE REVIEW

File: parm/soca/obsproc/obsproc_config.yaml [hunk 1]
```diff
@@ -12,14 +12,30 @@ observations:
...
     obsproc subdir: icec
-    output file: icec_amsr2_north_1.ioda.nc
+    output file: icec_amsr2_north.ioda.nc
     obsproc regex: AMSR2-SEAICE-NH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc
+- obs space:
+    name: icec_amsr2_south
+    provider: AMSR2
+    obsproc subdir: icec
+    output file: icec_amsr2_south.ioda.nc
+    obsproc regex: AMSR2-SEAICE-NH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc
```
Comment (ShastriPaturi) [inline parm/soca/obsproc/obsproc_config.yaml]: Should be:
obsproc regex: AMSR2-SEAICE-SH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc

File: test/soca/gw/prepdata.sh [hunk 2]
```diff
@@ -16,23 +16,26 @@ cdl2nc4() {
...
+        ${cdldir}/20180415095000-STAR-L3U_GHRSST-SSTsubskin-AVHRRF_MB-ACSPO_V2.80-v02.0-fv01.0.cdl
+
+# Fake AMSR2 icec
+cdl2nc4 icec/AMSR2-SEAICE-NH_v2r2_GW1_s201804150835180_e201804151014170_c201804151052280.nc \
+        ${project_source_dir}/test/soca/testdata/icec_amsr2_north_1.cdl
+cdl2nc4 icec/AMSR2-SEAICE-NH_v2r2_GW1_s201804151014190_e201804151150170_c201804151309570.nc \
+        ${project_source_dir}/test/soca/testdata/icec_amsr2_north_2.cdl
+cdl2nc4 icec/AMSR2-SEAICE-SH_v2r2_GW1_s201804150835180_e201804151014170_c201804151052280.nc \
+        ${project_source_dir}/test/soca/testdata/icec_amsr2_south_1.cdl
+cdl2nc4 icec/AMSR2-SEAICE-NH_v2r2_GW1_s201804151014190_e201804151150170_c201804151309570.nc \
```
Comment (ShastriPaturi) [inline test/soca/gw/prepdata.sh]: Should be "SH" here as well unless done so purposefully.

**Review by ShastriPaturi — CHANGES_REQUESTED** [review]

File: test/soca/gw/prepdata.sh (hunk 2 above)
Comment (guillaumevernieres) [reply test/soca/gw/prepdata.sh]: nope, bugs! Thanks for checking @ShastriPaturi .

**Review by apchoiCMD — DISMISSED** [review]
As far as I know, look good!

**Review by ShastriPaturi — DISMISSED** [review]
Looks good

**Review by AndrewEichmann-NOAA — DISMISSED** [review]
looks good to me

**Comment by guillaumevernieres** [comment]
@ShastriPaturi , @apchoiCMD , @AndrewEichmann-NOAA , I just adjusted the fake dates for the rads adt to fall within the DA window.

**Review by AndrewEichmann-NOAA — APPROVED** [review]

**Pull request merged**